src/extension/app/store/app.js

```diff
@@ -962,15 +962,6 @@ export class AppStore {
     // update live
     const resp = await this.api.updateLive(path);
     if (resp) {
-      // bust client cache for live and production
```

Comment:
I don't believe any of this was doing anything unless it just so happened that the publish happened same origin. This logic is instead moved to the publish plugin.